src/test/java/school/redrover/NinaYumTest.java

```diff
@@ -0,0 +1,37 @@
+package school.redrover;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class NinaYumTest {
+    @Test
```

Comment:
```suggestion

    @Test
```
Тут лучше добавить пустую строку